Ethernet/W5100S/w5100s.h

```diff
@@ -1932,7 +1932,7 @@ void     WIZCHIP_WRITE_BUF(uint32_t AddrSel, uint8_t* pBuf, uint16_t len);
  * @sa getIR()
  */
 #define setIR(ir) \
-		WIZCHIP_WRITE(IR, (ir & 0xE0)) //peter 2016.11.07 unreachable interrupt bit added
```

Comment:
Because of the low 4-bits is auto clear type, this line don't need to change the code.